gramps/plugins/gramplet/gallery.py

```diff
@@ -49,7 +51,10 @@ def build_gui(self):
         Build the GUI interface.
         """
         self.image_list = []
-        self.top = Gtk.Box(spacing=3)
+        if self.direction == "Horizontal":
+            self.top = Gtk.Box(spacing=3)
+        else:
+            self.top = Gtk.VBox(spacing=3)
```

Comment:
This has been deprecated.  Use `Gtk.Box(spacing=3, orientation=Gtk.Orientation.VERTICAL)` instead.